src/download/linux.md

```diff
@@ -0,0 +1,191 @@
+echo "deb [signed-by=/usr/share/keyrings/prebuilt-mpr-archive-keyring.gpg] https://proget.makedeb.org prebuilt-mpr $(lsb_release -cs)" | sudo tee /etc/apt/sources.list.d/prebuilt-mpr.list
+sudo apt update
+sudo apt install prismlauncher
+```
+
+NOTE: Prebuilt MPR only officially supports Debian 11, Ubuntu 20.04 and Ubuntu 22.04.
+
+## Installing with an MPR helper
+
+Installing UNA
+
+```bash
+bash <(curl -fsL https://github.com/AFK-OS/una/raw/main/install.sh)
+
+una update
+```
+
+Installing Prism Launcher
+
+```bash
+# stable source package:
+una install prismlauncher
+# stable binary package:
+una install prismlauncher-bin
+# latest git package:
+una install prismlauncher-git
+```
+
+You can replace una install with your preferred [MPR helper's](https://docs.makedeb.org/using-the-mpr/list-of-mpr-helpers/) install command.
+</div>
+
+<!--
+<div class="infobox top">
+
+# <img src="https://www.vectorlogo.zone/logos/getfedora/getfedora-icon.svg" height="20"> Fedora
+
+Two RPM packages are available on [Copr](https://copr.fedorainfracloud.org/coprs/sentry/prismlauncher/).
+
+```bash
+# enables the copr repo
+sudo dnf copr enable sentry/prismlauncher
+# stable releases
+sudo dnf install prismlauncher
+# nightly releases (git master)
+sudo dnf install prismlauncher-nightly
+```
+
+</div>
+<div class="infobox top">
+
+# <img src="https://www.gentoo.org/assets/img/logo/gentoo-signet.svg" height="20" /> Gentoo
+
+Ebuilds are available in the official Gentoo repository, under [`games-action/prismlauncher`](https://packages.gentoo.org/packages/games-action/prismlauncher).
+Note that, for the time being, it is not stabilized, so it's masked for `~amd64` only.
+
+```bash
+sudo emaint sync -a
+
+# If you need to unmask the package, and considering `package.accept_keywords` to be a folder.
+echo ">=games-action/prismlauncher-1.1.1" | sudo tee -a /etc/portage/package.accept_keywords/prismlauncher
+# Or do this if you want to build from the latest commit instead of a release
+echo "=games-action/prismlauncher-9999 **" | sudo tee -a /etc/portage/package.accept_keywords/prismlauncher
+
+emerge games-action/prismlauncher
+```
+
+Old ebuilds, as well as additional information, can be found [in the old prismlauncher overlay repository](https://gitlab.com/flowln/prismlauncher-gentoo/). Have fun! :)
+</div>
+--!>
+
+<div class="infobox top">
+
+# <img src="https://www.vectorlogo.zone/logos/nixos/nixos-icon.svg" height="20" /> Nix
+
+A [Nix derivation](https://github.com/PrismLauncher/PrismLauncher/blob/develop/nix/NIX.md) is available.
+</div>
+
+<!--
+<div class="infobox top">
+
+# <img src="https://upload.wikimedia.org/wikipedia/commons/d/d0/OpenSUSE_Logo.svg" height="20"> openSuse
+
+Prism Launcher is available on the [game:tools](https://software.opensuse.org/download.html?project=games%3Atools&package=prismlauncher) community repository.
```

Comment:
Currently not included in the offical repo, but is avaible in the same copr repo as the fedora package